lldb/source/ValueObject/DILEval.cpp

```diff
@@ -272,4 +272,66 @@ Interpreter::Visit(const UnaryOpNode *node) {
       m_expr, "invalid ast: unexpected binary operator", node->GetLocation());
 }
 
+llvm::Expected<lldb::ValueObjectSP>
+Interpreter::Visit(const MemberOfNode *node) {
+  Status error;
```

Comment:
Declare it where it's being used to make it clear what's its scope and contents.

Reply:
Done.